modules/andes-core/broker/src/main/java/org/wso2/andes/mqtt/MQTTLocalSubscription.java

```diff
@@ -235,6 +235,11 @@ public long getSubscribeTime() {
         return 0;
     }
 
+    @Override
+    public String getProtocolQueueName() {
+        return wildcardDestination;
```

Comment:
Can this lead to the registry resource getting deleted when we have multiple subscribers with the same wildcard destination? Since all the non-durable topic subscribers have the same protocol queue name in above situation, when only one subscription disconnects, we will be deleting the registry resource for other active subscriptions.